### PR TITLE
Try to slim down Python requirements

### PR DESCRIPTION
**Pull request by cclauss**
> **Description**: What does this PR achieve? [feature|hotfix|refactor]

Try to remove unnecessary Python dependencies which are being loaded in our __requirements*.txt__ files.  This process takes the work done in #1771 combined with [what Python imports we make](https://github.com/cclauss/What-are-my-Python-imports/blob/master/output.txt) to try to remove unused dependencies.   The command __grep -r <module_name> .__ was also run to see if each  module was referred to anywhere in the repo.  A conservative approach was taken on modules like __ipython__ so perhaps further slimming is possible.

> **Technical**: What should be noted about the implementation?

> **Testing**: Steps for reviewer to reproduce / verify this PR fixes the problem?

> **Evidence**: If this PR touches UI, please post evidence (screenshot) of it behaving correctly:

### CODE REVIEW

File: requirements_common.txt [hunk 1]
```diff
@@ -7,11 +7,10 @@ gunicorn==19.9.0
 iptools==0.6.1; python_version < '3.0'
 internetarchive==1.8.1
 lxml==4.2.5
-pyflakes==2.0.0
 pymarc==3.1.11
 python-memcached==1.59
 simplejson==3.16.0
-supervisor==3.0a12; python_version < '3.0'
+# supervisor==3.0a12; python_version < '3.0'
```
Comment (mekarpeles) [inline requirements_common.txt]: this should be moved to `olsystem`
cc: @hornc

File: requirements.txt [hunk 2]
```diff
@@ -1,35 +1,35 @@
 -r requirements_common.txt
 
-Fabric==1.10.1; python_version < '3.0'
+# Fabric==1.10.1; python_version < '3.0'
```
Comment (mekarpeles) [inline requirements.txt]: another one to move to `olsystem`

**Comment by hornc** [comment]
Fabric and supervisor have already been removed.

It'd be nice to remove the rest of the unnecessary / indirect requirements from these lists. I'll look into refreshing this

**Comment by hornc** [comment]
@cclauss can we close this in favour of moving forward with the 'radical' version #1773 ?
After comparing them both I think removing `requirements-common` is reasonable, and it looks to me like a lot of indirectly-required modules have made their way into requirements in the past, which explains why the file is so noisy now?

**Comment by cclauss** [comment]
Closing in favor of #1773

**Pull request closed without merging**